docs/source/installation.rst

```diff
@@ -2,19 +2,213 @@
 Installation
 ============
 
-BentoML is distributed as a Python package available `on PyPI <https://pypi.org/project/bentoml/>`_.
+Install 🍱 BentoML with your favourite package manager, alongside with whichever deep learning
+library you are working with, and you are ready to go!
+
+🍱 BentoML is distributed as a Python Package available `on PyPI <https://pypi.org/project/bentoml/>`_.
 
 * BentoML supports Linux/UNIX, Windows, and MacOS.
 * BentoML requires Python 3.7 or above.
 
+.. code-block::
+
+   pip install --user bentoml
+
+
+Install using virtual environment
```

Comment:
We may not need to educate the user to install with a virtual environment. BentoML should just be installed to their pre-existing environment where they do model training and development.

Reply:
I take a look at a bunch of installation guide from different pypi project, and seems that a lot of them recommend to use virtualenv, which is a good practice. But we feel that this is not needed, we can remove it.

Reply:
I think we shouldn't; Pytorch / Tensorflow don't, and I think a mixed environment is honestly worse and possibly confusing.

Do you have specific projects in mind here?

Reply:
https://spacy.io/usage/#section-installation https://huggingface.co/docs/transformers/installation#install-with-pip